_includes/v22.2/start-in-docker/mac-linux-steps.md

```diff
@@ -182,28 +187,28 @@ Now that your cluster is live, you can use any node as a SQL gateway. To test th
     (1 row)
     ~~~
 
-    As you can see, node 1 and node 2 behaved identically as SQL gateways.
+    As you can see, `roach1` and `roach2` perform identically as SQL gateways.
 
-5. Exit the SQL shell on node 2:
+5. Exit the SQL shell on `roach2`:
 
     {% include_cached copy-clipboard.html %}
     ~~~ sql
     > \q
     ~~~
 
-## Step 4. Run a sample workload
+### Step 5. Run a sample workload
 
 CockroachDB also comes with a number of [built-in workloads](cockroach-workload.html) for simulating client traffic. Let's run the workload based on CockroachDB's sample vehicle-sharing application, [MovR](movr.html).
 
-1. Load the initial dataset:
+1. Load the initial dataset on `roach1`:
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
     $ docker exec -it roach1 ./cockroach workload init movr \
     'postgresql://root@roach1:26257?sslmode=disable'
     ~~~
 
-2. Run the workload for 5 minutes:
+2. Run the workload for five minutes:
```

Comment:
Just noting that our docs are not consistent on this. Do we have a style note or issue somewhere?

Reply:
I couldn't find one. Where this has been defined in a style guide, it's usually numerals under 10 that you spell out.